### DISCUSSION TITLE
请问商品描述里面有空格吗

### DISCUSSION THREAD

**Discussion by huanhuan6666** [liusy58/NJU_AdvancedProgramming_SP22#115] [Q&A] [answered]
如题，如果带空格的话解析生成的SQL指令直接按照空格分割就不行了，变得很麻烦。想问问描述里需要支持空格吗？或者说可以报错不允许。

**Comment by binbin00** [comment, marked as answer]
https://github.com/liusy58/NJU_AdvancedProgramming_SP22/discussions/111
